certbot/display/util.py

```diff
@@ -253,9 +253,16 @@ def _return_default(self, prompt, default, cli_flag, force_interactive):
         :rtype: bool
 
         """
-        assert_valid_call(prompt, default, cli_flag, force_interactive)
+        # assert_valid_call(prompt, default, cli_flag, force_interactive)
         if self._can_interact(force_interactive):
             return False
+        elif default is None:
```

Comment:
Although info about `--force-interactive` is given in the warning in `_can_interact`.